### PR TITLE
Feature/user classes and permissions

### LINKED ISSUE #89: [WIP] Manage user classes and permissions
Provide an interface for managing site wide classes, a user's classes and permissions

### PR DESCRIPTION
**Pull request by ghost**
Closes #89 

Still a WIP

### CODE REVIEW

File: packages/gazelle-lib/package.js [hunk 1]
```diff
@@ -23,7 +23,9 @@ Package.onUse(function (api) {
     'maxharris9:classnames',
     'fourseven:scss',
     'accounts-ui',
-    'useraccounts:core'
+    'useraccounts:core',
```
Comment (salas-wcd) [inline packages/gazelle-lib/package.js:26]: You added it two times.

File: packages/gazelle-classes/package.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
+Package.describe({
+  name: 'gazelle-classes',
```
Comment (rhomes-wcd) [inline packages/gazelle-classes/package.js:2]: Follow the 'meteor-gazelle:[package name]' pattern.

**Comment by ghost** [comment]
@ajax-wcd This hasn't been updated in a while, what's going on with it?

**Comment by ghost** [comment]
Closing this PR as it has been umbrella'd in under #183

**Pull request closed without merging**